apps/mull-api/src/app/channel/channel.resolver.spec.ts

```diff
@@ -4,39 +4,36 @@ import { MockType } from '../user/user.service.spec';
 import {
   mockAllChannels,
   mockAllDirectMessageChannels,
-  mockCreateDmChannel,
+  mockAllEventChannels,
   mockCreateEventChannel,
 } from './channel.mockdata';
 import { ChannelResolver } from './channel.resolver';
 import { ChannelService } from './channel.service';
+import { CreateEventChannelInput } from './inputs/channel.input';
 
 const mockChannelService = () => ({
   getChannel: jest.fn((channelId: number) => {
     return mockAllChannels.find((channel) => channel.id === channelId);
   }),
-  getChannelByEvent: jest.fn((eventId, channelName, userId) => {
-    return mockAllChannels.find(
+  getChannelByEventId: jest.fn((eventId, channelName, userId) => {
+    return mockAllEventChannels.find(
       (channel) =>
         channel.event.id === eventId &&
         channel.name === channelName &&
         channel.event.host.id === userId
     );
   }),
-  createChannel: jest.fn((mockChannelData: CreateChannelInput) => ({ ...mockChannelData })),
-  getEventChannel: jest.fn((channelId: number) => {
-    return mockAllEventChannels.find((channel) => channel.id === channelId);
-  }),
   getDmChannel: jest.fn((channelId: number) => {
     return mockAllDirectMessageChannels.find((channel) => channel.id === channelId);
   }),
   createEventChannel: jest.fn((mockEventChannelData: CreateEventChannelInput) => ({
     ...mockEventChannelData,
   })),
-  createDmChannel: jest.fn((mockChannelData: CreateDmChannelInput) => ({ ...mockChannelData })),
+  createDmChannel: jest.fn().mockReturnValue(mockAllDirectMessageChannels[0]),
```

Comment:
Can we implement the logic instead of hard coding the return value? We've done this in other tests and it's made the debugging process/reusability of these mock implementations difficult. If it doesn't take too much time.